h/static/scripts/group-forms/utils/api.ts

```diff
@@ -45,15 +45,24 @@ export class APIError extends Error {
 /* Make an API call and return the parsed JSON body or throw APIError. */
 export async function callAPI(
   url: string,
-  method: string = 'GET',
-  json: object | undefined,
+  {
+    method = 'GET',
+    json = null,
+    headers = {},
+  }: {
+    method?: string;
+    json?: object | null;
+    headers?: any;
+  } = {},
```

Comment:
Changed this to an options object, otherwise you can't pass `headers` without also passing `json` or vice-versa.